.env.example

```
@@ -27,7 +27,8 @@ GATE_TRIGGER_CAPTURE_SOURCE=rtsp://127.0.0.1:8554/clear
 GATE_TRIGGER_CAPTURE_TIMEOUT_SECONDS=3
 # Wait for the vehicle to stop at the gate, then grab a short series of frames.
 GATE_TRIGGER_CAPTURE_DELAY_SECONDS=1.5
-GATE_TRIGGER_CAPTURE_COUNT=2
+GATE_TRIGGER_CAPTURE_HOT_KEYFRAMES=true
+GATE_TRIGGER_CAPTURE_COUNT=3
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GATE_TRIGGER_CAPTURE_COUNT key should go before the GATE_TRIGGER_CAPTURE_DELAY_SECONDS key

(UnorderedKey)